src/components/notification/NotificationBadge.tsx

```diff
@@ -0,0 +1,78 @@
+import { Intent, Popover, PopoverInteractionKind, Position, Tag } from '@blueprintjs/core';
+import * as React from 'react';
+import { AcademyNotification, AcademyNotificationType } from './notificationShape';
+
+type OwnProps = {
+  className?: string;
+  enableHover?: boolean; // enable or disable hover popover option
+  large?: boolean; // enable to use large style
+  notifications: AcademyNotification[];
```

Comment:
Why is this an array?

Reply:
The badge will display the number of relevant notifications, hence the array.